src/layers/BitmapLayer.js

```diff
@@ -169,7 +167,8 @@ BitmapLayer.defaultProps = {
   ...defaultProps,
   // We don't want this layer to bind the texture so the type should not be `image`.
   image: { type: 'object', value: {}, compare: true },
-  photometricInterpretation: { type: 'string', value: 'RGB', compare: true }
+  photometricInterpretation: { type: 'string', value: 'RGB', compare: true },
```

Comment:
hmm, I'm pretty sure this is a number?

```
PHOTOMETRIC_INTERPRETATIONS.RGB === 2
```

the switch/case statement matches on the values not the string keys.

Reply:
Fixed in the other PR with the defaults, thanks.